test/MetadataUtility.Tests/Metadata/WildlifeAcoustics/WaveHeaderExtractorTests.cs

```diff
@@ -0,0 +1,68 @@
+// <copyright file="WaveHeaderExtractorTests.cs" company="QutEcoacoustics">
+// All code in this file and all associated files are the copyright and property of the QUT Ecoacoustics Research Group.
+// </copyright>
+
+namespace MetadataUtility.Tests.Metadata
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using FluentAssertions;
+    using MetadataUtility.Audio;
+    using MetadataUtility.Metadata;
+    using MetadataUtility.Models;
+    using MetadataUtility.Tests.TestHelpers;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class WaveHeaderExtractorTests : TestBase
+    {
+        private readonly FilenameExtractor subject;
+
+        public WaveHeaderExtractorTests(ITestOutputHelper output)
+            : base(output)
+        {
+            this.subject = new FilenameExtractor(
```

Comment:
You probably actually want to test the WaveHeaderExtractor and not the file name extractor